src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/person/Person.kt

```diff
@@ -141,6 +141,9 @@ data class Person(
       val references = listOf(
         Reference.from(IdentifierType.CRO, prisoner.cro?.toString()),
         Reference.from(IdentifierType.PNC, prisoner.pnc?.toString()),
+        Reference.from(IdentifierType.NATIONAL_INSURANCE_NUMBER, prisoner.identifiers.find { it.type == "NINO" }?.value),
```

Comment:
Could add a helper function like the person getType for identifiers for the prisoner references e.g.
```
fun List<Reference>.getType(type: IdentifierType): List<Reference> {
      return this.filter { it.identifierType == type }
    }
```